src/brevitas/core/scaling/standalone.py

```diff
@@ -253,9 +248,11 @@ def forward(self, x: Tensor, threshold: Optional[Tensor] = None) -> Tensor:
         if threshold is None:
             threshold = torch.ones(1).type_as(x)
         if self.init_done:
-            threshold = self.stats_scaling_impl.restrict_clamp_threshold(
-                self.restrict_threshold_pre(threshold))
-            value = abs_binary_sign_grad(self.stats_scaling_impl.restrict_clamp_scaling(self.value))
+            threshold = self.stats_scaling_impl.restrict_threshold_pre(threshold)
```

Comment:
I would change this (and also below) to directly call `self.state_scaling_impl`.

Looks like we are doing all the same steps done within the forward pass, but one by one for no particular reason